lib/builderator/tasks/berkshelf.rb

```diff
@@ -23,6 +23,26 @@ def configure
         Interface.berkshelf.write
       end
 
+      desc 'chefignore', 'Create or Update chefignore to ignore .builderator directory'
+      def chefignore
+        ignore_file_list = ['.builderator']
```

Comment:
@athompson-r7 how about making this a `method_option` (`--ignore-files`) and defaulting to `[".builderator"]`?